src/components/menu/sub-menu-items.js

```diff
@@ -4,7 +4,7 @@ import MenuLink from './menu-link';
 const SubMenuItems = ({ page }) => {
   return (
     <ul>
-      {page.menuPageSubpages.map((item, i) => (
+      {page.subPages.map((item, i) => (
         <li key={item.id}>
           <MenuLink page={item} id={`sub-page-item-${i}`} />
```

Comment:
I only realized now that with `id={`sub-page-item-${i}`} `, we'll end up with multiple sub-menu items with the same IDs :sweat_smile: . And looks like we don't even need the id prop here? :thinking: 

I can make a separate PR to fix this if it's a hassle to fix that in this PR though : )

Reply:
I think that's a small change, so I'll do it 😊

Reply:
Awesome! Thanks a lot! : )